[profiler] update docs with repeat=1

davidberard98 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #95085

Specifying number of times to repeat is now required when defining the schedule.

davidberard98 committed:
[profiler] update docs with repeat=1

I think it may be required now to specify repeat=1

[ghstack-poisoned]

aaronenyeshi reviewed (approved):
LGTM! Thanks for adding this!

davidberard98 commented:
@pytorchbot merge

davidberard98 commented:
@pytorchbot rebase -s

davidberard98 committed:
Update on "[profiler] update docs with repeat=1"


Specifying number of times to repeat is now required when defining the schedule.

[ghstack-poisoned]

davidberard98 commented:
@pytorchbot merge
